### ISSUE TITLE
 Calling PropTypes validators directly is not supported by the `prop-types` 

### ISSUE THREAD

**Issue by ricking06** [malte-wessel/react-custom-scrollbars#153]
```
Warning: Failed prop type: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them. Read more at http://fb.me/use-check-prop-types
    in Scrollbars (created by Picker)
```
```
<Scrollbars ref="scroll" className="sscroll" onScroll={this.handleScroll.bind(this)}>
</Scrollbars>
```

**Comment by gaearon** [comment]
Please see https://github.com/reactjs/prop-types/issues/38.

This was a false positive caused by production version of `prop-types` generating a mistaken warning when development version of React is calling it.

**It is fixed in `prop-types@15.5.10`.**

You might need to rebuild the UMD bundle of the component that depends on `prop-types` (probably this repo) to see the changes.

**Comment by malte-wessel** [comment]
thanks @gaearon I added `prop-types@15.5.10` in `4.1.2`
